folsom/src/main/java/com/spotify/folsom/MemcacheClientBuilder.java

```diff
@@ -276,6 +277,28 @@ public MemcacheClientBuilder(final Transcoder<V> valueTranscoder) {
     return this;
   }
 
+  /**
+   * Specify the maximum number of operations that will be batched together in one network request.
+   *
+   * <p>If the client's batch-size is larger than your memcached server's value, you may experience
+   * an increase in `conn_yields` on your memcached server's stats...which indicates your server is
+   * switching to other I/O connections during the batch request to not starve other connections.
+   *
+   * <p>If this value is too low, your will make more network requests per operation, thus reducing
+   * your server's overall throughput.
+   *
+   * <p>The optimal value should be matched to your workload and roughly the same value as your
+   * memcached server's `-R` argument, which defaults to 20.
+   *
+   * @param batchSize the maximum number of operations per batched client request. Default is
+   *     {@value Settings#DEFAULT_BATCH_SIZE}.
+   * @return itself
+   */
+  public MemcacheClientBuilder<V> withRequestBatchSize(final int batchSize) {
+    this.batchSize = batchSize;
```

Comment:
We should probably also sanity check the value here (must be >= 1 at least)

Reply:
added param check